src/people.js

```diff
@@ -26,6 +26,29 @@
  * @returns {PeopleResponse}  - Processed information
 */
 
-function peopleInformation(people) { }
+function peopleInformation(people) { 
```

Comment:
Reduce would be awesome here as well. Use reduce whenever you are taking some data and 'reducing' it down to get other values